src/app.module.ts

```diff
@@ -29,12 +24,27 @@ const {
       isGlobal: true,
     }),
     RatesModule,
-    MongooseModule.forRoot(
-      `mongodb://${MONGODB_HOST}:${MONGODB_PORT}/tickersdb`,
-    ),
-    RedisModule.forRoot({
-      type: 'single',
-      url: `redis://${REDIS_HOST}:${REDIS_PORT}`,
+    MongooseModule.forRootAsync({
+      imports: [ConfigModule, LoggerModule],
+      inject: [ConfigService, Logger],
+      useFactory: (config: ConfigService, logger: Logger) => {
+        const port = config.get('server.mongodb.port');
+        const host = config.get('server.mongodb.host');
+
+        return {
+          uri: `mongodb://${host}:${port}/${MONGODB_NAME}`,
+          retryAttempts: 0,
```

Comment:
Lower the connection timeout. It takes about 30 seconds to throw an error and kill the process when using incorrect host/port.